### DISCUSSION TITLE
qfield error date display calendar

### DISCUSSION THREAD

**Discussion by Pipo1974** [opengisch/QField#4957] [Q&A]
The qfield calendar displays tomorrow's date by default, which generates an error when entering data. By default, it should display the date of the current day (today).
When applying the form in qgis it works correctly but when passing to qfield it is displayed with an error. Any idea how to solve it?

**Comment by beanzmo** [comment]
Hi @Pipo1974  - There could be multiple reasons for this but I presume this is not a specific problem with QField.

1. have you checked in your QGIS Project the settings of the attribute form?
2. Where are you based?

Indeed there is a bug with QField when it comes to the date widget. /issues/4069
Maybe you resolved it already?
